ctapipe/io/containers.py

```diff
@@ -611,3 +618,189 @@ class TimingParametersContainer(Container):
+    )
+    time_std = Field(
+        None,
+        "np array of signal time standard deviation (n_chan, n_pix)",
+        unit=u.ns
+
+    )
+    relative_gain_mean = Field(
+        None,
+        "np array of the relative flat-field coefficient mean (n_chan, n_pix)"
+    )
+    relative_gain_median = Field(
+        None,
+        "np array of the relative flat-field coefficient  median (n_chan, n_pix)"
+    )
+    relative_gain_std = Field(
+        None,
+        "np array of the relative flat-field coefficient standard deviation (n_chan, n_pix)"
+    )
+    relative_time_median = Field(
+        None,
+        "np array of time (median) - time median averaged over camera (n_chan, n_pix)",
+        unit=u.ns)
+
+    charge_median_outliers = Field(
+        None,
+        "Boolean np array of charge (median) outliers (n_chan, n_pix)"
+    )
+    time_median_outliers = Field(
+        None,
+        "Boolean np array of pixel time (median) outliers (n_chan, n_pix)"
+    )
+
+
+class PedestalContainer(Container):
+    """
+    Container for pedestal parameters obtained from a set of
+    [n_pedestal] pedestal events
+    """
+    n_events = Field(0, 'Number of events used for statistics')
+    sample_time = Field(0, 'Time associated to the pedestal event set', unit=u.s)
+    sample_time_range = Field(
+        [],
+        'Range of time of the pedestal events [t_min, t_max]',
+        unit=u.s
+    )
+    charge_mean = Field(
+        None,
+        "np array of pedestal average (n_chan, n_pix)"
+    )
+    charge_median = Field(
+        None,
+        "np array of the pedestal  median (n_chan, n_pix)"
+    )
+    charge_std = Field(
+        None,
+        "np array of the pedestal standard deviation (n_chan, n_pix)"
+    )
+    charge_median_outliers = Field(
+        None,
+        "Boolean np array of the pedestal median outliers (n_chan, n_pix)"
+    )
+    charge_std_outliers = Field(
+        None,
+        "Boolean np array of the pedestal std outliers (n_chan, n_pix)"
+    )
+
+
+class PixelStatusContainer(Container):
+    """
+    Container for pixel status information
+    It contains masks obtained by several data analysis steps
+    At r0 level only the hardware_mask is initialized
+    """
+    hardware_mask = Field(
+        None,
+        "Boolean np array (mask) from the hardware pixel status data (n_chan, n_pix)"
+    )
+
+    pedestal_mask = Field(
+        None,
+        "Boolean np array (mask) from the pedestal data analysis (n_chan, n_pix)"
+    )
+
+    flatfield_mask = Field(
+        None,
+        "Boolean np array (mask) from the flat-flield data analysis (n_chan, n_pix)"
+    )
+
+
+class WaveformCalibrationContainer(Container):
+    """
+    Container for the pixel calibration coefficients
+    """
+    time = Field(0, 'Time associated to the calibration event', unit=u.s)
+    time_range = Field(
+        [],
+        'Range of time of validity for the calibration event [t_min, t_max]',
+        unit=u.s
```

Comment:
How is the time defined here? I guess seconds since the start of the observation? Should we make that an absolute time?

Reply:
This is for the moment the event.r1.tel[tel_id].trigger_time 
If this must be used as timestamp for retrieving calibration data, it should be indeed better an absolute time (or at least the time reference should be easily accessible)